omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PatientResource1_8.java

```diff
@@ -225,6 +226,7 @@ public Object create(SimpleObject propertiesToCreate, RequestContext context) th
 	}
 	
 	public Patient getPatient(SimpleObject propertiesToCreate) {
+		removeKeys(propertiesToCreate, "preferredName", "preferredAddress", "deathdateEstimated", "birthtime");
```

Comment:
The framework should be ignoring these, if it doesn't we need to fix it instead of doing it here

Reply:
deathdateEstimated and birthtime were added like in 1.11 and 1.10 respectively, see my comments below, meaning you should not be worrying about them here since this is a 1.8 resource, it is just the test data that is incorrect

Reply:
Removing of preferredName and preferredAddress is the responsibility of the client and not the rest API